simulator/mips/t/unit_test.cpp

```diff
@@ -71,98 +40,44 @@ TEST_CASE( "MIPS32_instr: Divmult")
     CHECK(MIPS32Instr("ddiv").is_divmult());
```

Comment:
Please move these checks to the per-instruction tests if they exist.
```c++
TEST_CASE( "MIPS32_Instr: mult ... ")
{
// ...
  CHECK( instr.is_divmult());
}
```

Reply:
Done.